specification/paths/Suggest-hscode.json

```diff
@@ -0,0 +1,104 @@
+{
+  "post": {
+    "tags": [
+      "RPC"
+    ],
+    "security": [
+      {
+        "OAuth2": [
+          "landed_cost"
+        ]
+      }
+    ],
+    "summary": "Get hs code suggestion.",
+    "description": "This endpoint returns the hs code suggestions for the supplied product.",
+    "requestBody": {
+      "description": "The required fields for the suggestion",
+      "required": true,
+      "content": {
+        "application/json": {
+          "schema": {
+            "type": "object",
+            "required": [
+              "data"
+            ],
+            "properties": {
+              "data": {
+                "required": [
+                  "clientSKU",
+                  "clientDescription"
+                ],
+                "properties": {
+                  "clientSKU": {
+                    "type": "string"
+                  },
+                  "clientDescription": {
+                    "type": "string"
+                  },
+                  "sourceLanguage": {
+                    "type": "string"
+                  },
+                  "sourceCountry": {
+                    "type": "string"
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "responses": {
+      "200": {
+        "description": "Landed costs have been added to the shipment items and customs.",
+        "content": {
+          "application/json": {
+            "schema": {
+              "type": "object",
+              "required": [
+                "data"
+              ],
+              "properties": {
+                "data": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "required": [
+                      "hurricaneUUID",
+                      "hsCode",
+                      "commodityCode",
+                      "vectorWeight",
+                      "productDescription"
```

Comment:
Again, here we should stick as close as possible to field names we already have in the specs. These values will be used when creating shipments so field names should be available to choose from.